.markdownlint.json

```diff
@@ -0,0 +1,19 @@
+{
+  "MD001": false,
+  "MD004": false,
+  "MD009": false,
+  "MD013": false,
+  "MD022": false,
+  "MD024": false,
+  "MD025": false,
+  "MD026": false,
+  "MD029": false,
+  "MD033": false,
+  "MD034": false,
+  "MD036": false,
+  "MD040": false,
+  "MD041": false,
+  "MD042": false,
+  "MD045": false,
+  "MD046": false
```

Comment:
These are mysterious. Does this tool support a yaml configuration file so that comments can be added to describe each one?

Reply:
I believe so! Stay tuned.

Reply:
done ✅